lib/brains/webhooks.py

```diff
@@ -30,6 +35,29 @@ def process(self):
 
         log.info('Processing event: {0}'.format(self.webhook.kind))
         method()
+        self.processed = True
+
+    @property
+    def data(self):
+        if not self.processed:
+            return
+
+        subscription = self.get_subscription()
+        try:
+            transaction = self.transactions[0]
```

Comment:
Maybe just add a comment here that we're only getting the most recent one. Otherwise the code looks awkward.